doc/sphinx-guides/source/installation/external-tools.rst

```diff
@@ -59,6 +62,15 @@ To list all the external tools that are available in Dataverse:
 
 ``curl http://localhost:8080/api/admin/externalTools``
 
+Showing an External Tool in Dataverse
```

Comment:
What's the purpose / need for this API? (besides having it in a test?)

Reply:
It's for sysadmins to check which external tools have been installed.

Reply:
Sure, for curl http://localhost:8080/api/admin/externalTools, which lists the tools, I'm asking specifically about this new one that takes an id. i.e if you have the id already then you already know about the tool. Doesn't seem to add any value, but maybe I'm missing something.

Reply:
I can imagine a somewhat paranoid sysadmin or tool maker adding a tool and then wanting to double check that correct values were saved in the database. For example, here I'm asserting that the tool that was just added is a dataset level tool but you could make assertions on other things:

        Response getTool = UtilIT.getExternalTool(id);
        getTool.prettyPrint();
        getTool.then().assertThat()
                .body("data.scope", CoreMatchers.equalTo("dataset"))
                .statusCode(OK.getStatusCode());

To me, this is all basic CRUD stuff. We're doing a read of a single item. I meant to add this back when I first implemented external tools but forgot about it. I'm trying to correct an oversight, something I forgot to add.